Plane: Unify tilt servo names

Hwurzburg opened the pull request:
@tridge per our conversation yesterday

khancyr reviewed (changes requested):
41:TiltMotor,45:TiltMotorRear  -->  41:TiltMotorsFront,45:TiltMotorsRear
75:TiltMotorLeft,  -->  75:TiltMotorFrontLeft
76:Tilt MotorRight  --> 76:Tilt MotorFrontRight

Shouldn't the space on 76 be removed to be consistent with 75:TiltMotorFrontLeft ?

Hwurzburg commented:
> 41:TiltMotor,45:TiltMotorRear --> 41:TiltMotorsFront,45:TiltMotorsRear
> 75:TiltMotorLeft, --> 75:TiltMotorFrontLeft
> 76:Tilt MotorRight --> 76:Tilt MotorFrontRight
> 
> Shouldn't the space on 76 be removed to be consistent with 75:TiltMotorFrontLeft ?

yes...just switched to ATOM editor and have trouble seeing spaces at the end of a line wrap, wish it would highlight the ending white spaces when it wraps a line...fixed...thanks

khancyr reviewed (commented):
From a syntax point of view that is OK.
But I got no idea if the changes are right or not
